addons/mail/static/src/models/rtc_call_participant_card/rtc_call_participant_card.js

```diff
@@ -111,7 +111,7 @@ function factory(dependencies) {
          * @returns {boolean}
          */
         _computeIsTalking() {
-            return Boolean(this.rtcSession && this.rtcSession.isTalking && !this.rtcSession.isMuted);
+            return Boolean(this.rtcSession && this.rtcSession.isTalking && !this.rtcSession.isMutedOrDeaf);
```

Comment:
why is this one not depending on audioTrack like on the other file?

Reply:
`rtcSession.isTalking` peers don't send the `isTalking` signal if they don't have a track.